kernel/packages/decentraland-ecs/types/@decentraland/RestrictedActions/index.d.ts

```diff
@@ -0,0 +1,24 @@
+declare module '@decentraland/RestrictedActions' {
+  /**
+   * move player to a position inside the scene
+   *
+   * @param position PositionType
+   * @param cameraTarget PositionType
+   */
+  export function movePlayerTo(newPosition: PositionType, cameraTarget?: PositionType): Promise<void>
+
+  export type PositionType = { x: number; y: number; z: number }
+
+  /**
+   * trigger an emote on the current player
+   *
+   * @param emote the emote to perform
+   */
+  export function triggerEmote(emote: Emote): Promise<void>
+
+  export type Emote = {
+    predefined: PredefinedEmote
+  }
+
+  export type PredefinedEmote = 'WAVE' | 'FIST_PUMP' | 'ROBOT' | 'RAISE_HAND' | 'CLAP' | 'MONEY' | 'KISS'
```

Comment:
This can't be an enum, because we are using RPC here, and it is not supported

Reply:
but the enum could be used with the items indexes as usual right? can't we use ints in the RPC?

Reply:
Regular enums won't work on RPS, regardless of whether they are mapped to ints or strings. But your comment gave the idea to try const enums, and they work!!

Thanks @pravusjif !

cc @menduz